components/butterfly/src/member.rs

```diff
@@ -131,14 +131,28 @@ pub struct Member {
 }
 
 impl Member {
-    /// Returns the socket address of this member.
+    /// Returns the swim socket address of this member.
     ///
     /// # Panics
     ///
     /// This function panics if the address is un-parseable. In practice, it shouldn't be
     /// un-parseable, since its set from the inbound socket directly.
     pub fn swim_socket_address(&self) -> SocketAddr {
-        let address_str = format!("{}:{}", self.get_address(), self.get_swim_port());
+        self.socket_address(self.get_swim_port())
+    }
+
+    /// Returns the gossip socket address of this member.
+    ///
+    /// # Panics
+    ///
+    /// This function panics if the address is un-parseable. In practice, it shouldn't be
+    /// un-parseable, since its set from the inbound socket directly.
```

Comment:
Parsable is the word you're looking for I think ;) (at least according to [dictionary.com](http://www.dictionary.com/browse/parsable))

Reply:
Heh, it's a copy-pasta from the function above it. Will fix both.